entities/capitalist_card.rb

```diff
@@ -0,0 +1,49 @@
+class CapitalistCard < BaseCard
+  def initialize
+    super
```

Comment:
you don't have an `initialize` method in BaseCard class, so no need to call super

Reply:
Yes, I don't need, but ```rubocop``` needs. Deleted ```super``